packages/devtools_app/lib/src/screens/debugger/codeview_controller.dart

```diff
@@ -269,6 +340,9 @@ class ParsedScript {
 
   final Set<int> executableLines;
 
+  final Set<int> hitLines;
+  final Set<int> missedLines;
```

Comment:
super nit: should these be named `coverageHitLines` and `coverageMissedLines` to make it clear what these are for? hit and miss by themselves may be confusing